tenant/src/main/java/org/folio/spring/tenant/service/SchemaService.java

```diff
@@ -1,17 +1,24 @@
 package org.folio.spring.tenant.service;
 
+import java.util.regex.Pattern;
 import org.folio.spring.tenant.properties.BuildInfoProperties;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
 
 @Service
 public class SchemaService {
 
+  private static final Pattern SCHEMA_REGEXP = Pattern.compile("[a-zA-Z0-9_]+");
+
   @Autowired
   private BuildInfoProperties buildInfoProperties;
 
   public String getSchema(String tenant) {
-    return String.format("%s_%s", tenant, buildInfoProperties.getArtifact()).replace("-", "_");
+    var schema = String.format("%s_%s", tenant, buildInfoProperties.getArtifact()).replace("-", "_");
```

Comment:
I do not think weak typings have been introduced in the project yet. Should we consider deferring that? Otherwise, changes are much appreciated and will approve.